tools/check_release.json

```diff
@@ -0,0 +1,16 @@
+{
+  "config" : {
+    "mbed_repo_path" : "C:/Users/annbri01/Work/Mercurial"
```

Comment:
The point of the config is it allows any path / user to use the script. So by modifying the version of this it could run in CI or in a general account.